packages/clay-navigation/src/BreadcrumbEllipsis.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+import {IBreadcrumbItem} from './BreadcrumbItem';
+
+export interface IBreadcrumbEllipsisProps {
+	/**
+	 * Property to define BreadcrumbEllipsis's items.
+	 */
+	items?: Array<IBreadcrumbItem>;
```

Comment:
Is this really an optional param? Meaning, would we ever expect or want this component not to receive items?

Reply:
I left this optional due I was initializing with an empty array and I was taking as a base PaginationEllipsis that this property is defined as optional. Perhaps, make sense turning this one obligatory.